frontend/src/pages/mentoringCreate/components/CertificateInput/CertificateInput.tsx

```diff
@@ -0,0 +1,225 @@
+import styled from '@emotion/styled';
+
+import certificateUploadIcon from '../../../../common/assets/images/certificateUploadIcon.svg';
+import deleteIcon from '../../../../common/assets/images/deleteIcon.svg';
+import downIcon from '../../../../common/assets/images/downIcon.svg';
+import usePreviewImage from '../../../../common/hooks/usePreviewImage';
+
+interface CertificateInputProps {
+  onDeleteButtonClick: () => void;
+}
+
+function CertificateInput({ onDeleteButtonClick }: CertificateInputProps) {
+  const { previewUrl, handleImageChange } = usePreviewImage();
+  return (
+    <StyledContainer>
+      <StyledTitleWrapper>
+        <h4>자격증</h4>
+        <button type="button" onClick={onDeleteButtonClick}>
+          <img src={deleteIcon} alt="삭제 아이콘" />
+        </button>
+      </StyledTitleWrapper>
+      <StyledContentWrapper>
+        <p>유형</p>
+        <select defaultValue="자격증" name="certificateType">
+          <option value="자격증">자격증</option>
+          <option value="학력">학력</option>
+          <option value="수상 경력">수상 경력</option>
+          <option value="기타">기타</option>
+        </select>
```

Comment:
p5: 나중에 서버에서 데이터를 가지고 오게 되면, 자격증, 학력, 수상 경력, 기타 를 하나의 데이터로 만든 다음에 map을 써서 만들면 관리하기 편할 거 같아 👍

```suggestion
const CERTIFICATE_TYPE_LIST = ['자격증','학력','수상 경력','기타'] as const
...
<select defaultValue ={} name="certificateType">
     {CERTIFICATE_TYPE_LIST.map((option, index) => (
        <option key={index} value={option}>
          {option}
        </option>
      ))}
</select>
```